automated install/basic-install.sh

```diff
@@ -1584,6 +1690,8 @@ stop_service() {
     printf "  %b %s..." "${INFO}" "${str}"
     if is_command systemctl ; then
         systemctl stop "${1}" &> /dev/null || true
+    elif is_command rc-service ; then
+        rc-service "${1}" stop &> /dev/null || true
     else
         service "${1}" stop &> /dev/null || true
     fi
```

Comment:
These constructs would be better as `case`s.

Reply:
Something like this ?? O.o

```
case is_command systemctl in
  0)
    systemctl stop "${1}" &> /dev/null || true
    ;;
  *)
    case is_command rc-service in
      0)
        rc-service "${1}" stop &> /dev/null || true
        ;;
      *)
        service "${1}" stop &> /dev/null || true
        ;;
esac
```

Reply:
More along the lines of 

Run `is-command systemctl` `is-command rc-service` once at the start of the script and save the results to a variable like `INIT_SYSTEM`. Then do a `case` on `INIT_SYSTEM` with the variable contents as the cases.

```
if is systemctl; then INIT_SYSTEM=systemd
if is rc-service; then INIT_SYSTEM=openrc

case INIT_SYSTEM in
  systemd)
   blabla;
  openrc)
  blabla;
 *)
  blahbla
```